appConfig.js

```diff
@@ -35,6 +35,15 @@ const config = {
     url: 'https://www.nypl.org/books-music-dvds/recommendations',
     label: 'Recommendations',
   },
+  listMetaData: {
```

Comment:
Since we're not specifically using this for meta tag data, you can rename this to 'heroData' or maybe 'pickDescription'?

Reply:
I like `heroData`. It stay's consistent with `heroImageUrl`. I'll rename. Thanks!